packages/ds-react-core/tsconfig.build.json

```diff
@@ -7,7 +7,17 @@
     "declarationDir": "dist/types",
     "declarationMap": true,
     "sourceMap": true,
-    "skipLibCheck": false
+    "skipLibCheck": false,
+    "types": ["node", "react", "react-dom"]
```

Comment:
Types needed for testing have been added to the root-level `tsconfig.json`. 
The build tsconfig needs to specify only the types that are needed to actually build the design system, so it overrides the base config here.